packages/ui-components/src/components/UITable/UITable.tsx

```diff
@@ -620,6 +640,17 @@ export class UITable extends React.Component<UITableProps, UITableState> {
             errorMessage = column.validate(value);
         }
         if (editedCell && editedCell.errorMessage !== errorMessage) {
+            if (editedCell.rowIndex && editedCell?.column?.key) {
```

Comment:
this condition causes row 0 to be ignored
https://github.com/SAP/open-ux-tools/pull/869